src/IB/IBFEMethod.cpp

```diff
@@ -1115,8 +1145,6 @@ IBFEMethod::spreadFluidSource(const int q_data_idx,
         assertStructureOnFinestLevel();
         getScratchToPrimarySchedule(d_hierarchy->getFinestLevelNumber(), q_data_idx).fillData(data_time);
     }
-
-    return;
```

Comment:
?

Reply:
I've been starting to remove "unnecessary" `return` statements at the end of function bodies. (I used to include explicit `return` statements in all function bodies.) I can revert if you prefer to have these in all functions.

Reply:
I am ambivalent about the style but I would prefer to delete these all at once so that the style is consistent.

Reply:
I am not sure that the current style is very consistent at this point. However, if you want to revert this change, that is fine with me.